<launch>
    <param name="/use_sim_time" value="True"/>
    <arg name="real_time_factor" default="1"/>
    <arg name="config_path" default="$(find satellite_simulator)/config/reciever_config.yaml"/>
    <arg name="constellation_file" default="$(find satellite_simulator)/config/constellation.yaml"/>

    <arg name="gps_noise_model_folder" default="$(find satellite_simulator)/models"/>

    <param name="gps_noise_model_folder" value="$(arg gps_noise_model_folder)"/>

    <node name="gnss_simulator" pkg="satellite_simulator" type="gnss_simulator" output="screen">
        <param name="constellation_file" value="$(arg constellation_file)"/>
        <param name="config_path" value="$(arg config_path)"/>
    </node>

    <node name="publish_mp_nsat" pkg="satellite_simulator" type="gps_error_publisher.py" output="screen">
        <param name="config_path" value="$(arg config_path)"/>
    </node>
</launch>
